spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  context 'methods' do
+    it 'have the date_form_helper method?' do
+      expect(helper).to respond_to(:date_form_helper)
+    end
+
+    it 'have the current_user_id method?' do
+      expect(helper).to respond_to(:current_user_id)
+    end
+  end
+
+  describe '.date_form_helper' do
+    it 'returns a hash' do
+      expect(helper.date_form_helper).to eq(as: :string,
+                                            input_html: { class: :datepicker })
+    end
+  end
+
+  describe '.current_user_id' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.